### ISSUE TITLE
Gracefully handle missing images / invalid URLs / missing alt text - Stepper/Carousel React

### ISSUE THREAD

**Issue by ssabo1** [sparkdesignsystem/spark-design-system#1544]
**User Story:**
As Spark we started implementing the Carousel component already and need to complete the work. 

**Notes:**
- Bob start the initial work for React Carousel  
https://sparkdesignsystem.com/patterns/components/stepper 

**AC:**
- gracefully handle missing images, or invalid URLs, or missing alt text. 
- If any step has an imgSrc, then every step must have an imgSrc and an imgAlt.
